src/ssl.c

```diff
@@ -16776,7 +16796,28 @@ size_t wolfSSL_get_client_random(const WOLFSSL* ssl, unsigned char* out,
         return 0;
     }
 
+    #if defined(OPENSSL_EXTRA)
+    int wolfSSL_SHA_Transform(WOLFSSL_SHA_CTX* sha, 
```

Comment:
I don't see SHA_Transform as an OpenSSL API.
Should be defined to SHA1_Transform if needed.

Reply:
older version of OpenSSL, such as version 1.0.1, has [the API](https://github.com/openssl/openssl/blob/4675a56a3ce2bef715386e45036426c8a4329d39/crypto/sha/sha.h#L115)

Reply:
Thanks!